UnityProject/Assets/Scripts/PlayGroups/PlayerMove.cs

```diff
@@ -145,23 +165,43 @@ private Vector3Int GetMoveDirection(KeyCode action)
 			{
 				return Vector3Int.zero;
 			}
-
-			switch (action)
+			//This needs a refactor, but this way AZERTY will work without weird conflicts.
```

Comment:
You can use the following to add it to automatic TODO list in rider IDE:
//TODO This needs a refactor, but this way AZERTY will work without weird conflicts.